e2e/tests/home/home-page.spec.ts

```diff
@@ -1,63 +0,0 @@
-import BaseElement from 'app/element/base-element';
-import HomePage from 'app/page/home-page';
-import WorkspaceCard from 'app/component/card/workspace-card';
-import { signInWithAccessToken } from 'utils/test-utils';
-import { getStyleValue } from 'utils/element-utils';
-
-describe('Home page ui tests', () => {
-  beforeEach(async () => {
-    await signInWithAccessToken(page);
-  });
-
-  test('Check visibility of Workspace cards', async () => {
-    await checkCreateNewWorkspaceLink();
-
-    const allCards = await new WorkspaceCard(page).findAllCards();
-    let width;
-    let height;
-    for (const card of allCards) {
-      const cardElem = BaseElement.asBaseElement(page, await card.asElement());
-      expect(await cardElem.isVisible()).toBe(true);
-      const size = await cardElem.getSize();
-      expect(size).toBeTruthy();
-      expect(size.height).toBeGreaterThan(1);
-      expect(size.width).toBeGreaterThan(1);
-
-      if (width === undefined || height === undefined) {
-        width = size.width; // Initialize width and height with first card element's size, compare with rest cards
-        height = size.height;
-      } else {
-        expect(size.height).toEqual(height);
-        expect(size.width).toEqual(width);
```

Comment:
Is card size existence and equality something we're concerned will fail? I don't think this is a valuable test unless there's specific logic changing card sizes, which I don't think we have.